src/Components/Wizard/index.js

```diff
@@ -51,8 +51,12 @@ const ProvisioningWizard = ({ isOpen, onClose, ...props }) => {
 };
 
 ProvisioningWizard.propTypes = {
-  isOpen: PropTypes.bool,
+  isOpen: PropTypes.bool.isRequired,
   onClose: PropTypes.func,
+  image: PropTypes.shape({
+    name: PropTypes.string,
+    id: PropTypes.string,
+  }).isRequired,
```

Comment:
@Gundersanne does this make sense to you, or shall we get more options from IB? :)
I guess we will rething this once we are in multicloud anyway, so this is M0 options in my mind, but if you have ideas how to make it ready for M1 now, happy to talk it over :)